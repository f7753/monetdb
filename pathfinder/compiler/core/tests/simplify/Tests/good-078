#!/bin/bash

pf -s10 <<'EOT'
declare namespace foo = 'http://www.foo.bar/';

<foo:b/>
EOT
echo $?
