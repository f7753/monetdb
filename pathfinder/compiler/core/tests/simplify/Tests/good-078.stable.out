stdout of test 'good-078` in directory 'compiler/core/tests/simplify` itself:


# 13:01:11 >  
# 13:01:11 >  Mtimeout -timeout 60 ./good-078 good-078 
# 13:01:11 >  

0

# 13:01:11 >  
# 13:01:11 >  Done.
# 13:01:11 >  

